facebook-instant-articles.php

```diff
@@ -321,20 +314,49 @@ function inject_url_claiming_meta_tag() {
 		$publishing_settings = Instant_Articles_Option_Publishing::get_option_decoded();
 		$fb_page_settings = Instant_Articles_Option_FB_Page::get_option_decoded();
 
-		if ( isset( $fb_page_settings['page_id'] ) ) {
+		if ( isset( $fb_page_settings[ 'page_id' ] ) ) {
 			?>
-			<meta property="fb:pages" content="<?php echo esc_attr( $fb_page_settings['page_id'] ); ?>" />
+			<meta property="fb:pages" content="<?php echo esc_attr( $fb_page_settings[ 'page_id' ] ); ?>" />
 			<?php
 		}
 	}
 	add_action( 'wp_head', 'inject_url_claiming_meta_tag' );
 
+	/**
+	 * Automatically add meta tag for Instant Articles Open Publish scraper
+	 *
+	 * @since 4.0
+	 */
+	function inject_op_markup_meta_tag() {
+		$post = get_post();
+		// Transform the post to an Instant Article.
+		$adapter = new Instant_Articles_Post( $post );
+		$url = $adapter->get_canonical_url();
+		?>
+		<meta property="op:markup_url" content="<?php echo esc_attr( $url ); ?>?op=1" />
```

Comment:
@everton-rosario what's the `op` parameter used for?

We should use the [`add_query_arg()`](https://developer.wordpress.org/reference/functions/add_query_arg/) function here:

```
esc_attr( add_query_arg( array( 'op' => '1' ), $url ) );
```